osidb/models.py

```diff
@@ -1769,15 +1769,18 @@ def _validate_historical_affectedness_resolution(self):
                 or self.resolution != old_affect.resolution
             ):
                 raise ValidationError(
-                    f"{self.resolution} is not a valid resolution for {self.affectedness}."
+                    f"Affect ({self.uuid}) for {self.ps_module}/{self.ps_component} has an invalid "
+                    f"affectedness/resolution combination: {self.resolution} is not a valid resolution "
```

Comment:
Nitpick: says "affectedness/resolution" first, prints resolution/affectedness second.

```suggestion
                    f"resolution/affectedness combination: {self.resolution} is not a valid resolution "
```